migrations/2020-11-05-112238_initialize_table/down.sql

```diff
@@ -0,0 +1,9 @@
+-- This file should undo anything in `up.sql`
+DROP TYPE IF EXISTS Laporan_status;
```

Comment:
Move line 2 to line 10. Because its cause error "Rolling back migration 2020-11-05-112238_initialize_table
Executing migration script /home/bima/Projects/rust-bootcamp/numbering-engine/migrations/2020-11-05-112238_initialize_table/down.sql
Failed with: cannot drop type laporan_status because other objects depend on it"
when use command `diesel migration redo`

Reply:
betul ini harus drop dulu table laporannya baru bisa drop type nya

Reply:
betul betul betul.